grails-bootstrap/src/main/groovy/grails/util/Environment.java

```diff
@@ -659,10 +663,25 @@ public static boolean isReloadingAgentEnabled() {
         try {
             Class.forName("org.springframework.boot.devtools.RemoteSpringApplication");
             reloadingAgentEnabled = Environment.getCurrent().isReloadEnabled();
+            LOG.debug("Found spring-dev-tools on the class path");
+        }
+        catch (ClassNotFoundException e) {
+            reloadingAgentEnabled = false;
+        }
+        try {
```

Comment:
Shouldn't this try block be inside the previous catch? A previous true value could be overriden to false